_posts/2021-11-22-rse-survey.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: International RSE Survey
+tags: [survey]
+posted_by: Ian Cosden
+---
+
+
+As part of an international effort, the US-RSE has worked with RSE organizations in other countries to develop the 2021 RSE Survey. 
+You don’t need to be called an RSE to complete the survey.
+Your formal job title does not matter: we want to hear from you if you develop software for academic research as part of your work.
+The survey provides considerable detail about the RSE role: a person’s route into the role, the type of work they conduct, their working practices, job satisfaction and demographics.
+This means it takes around 20 minutes to complete.
+We appreciate that this is a significant amount of your time to donate - and we thank you for it! 
+
+We strongly encourage you to respond to the survey and to share it with others who may be interested.
+The results of this survey will be greatly beneficial to US-RSE as we move forward.
+The more responses we have, the better our knowledge.
+
+For more details on the survey visit https://software.ac.uk/news/complete-international-rse-survey-and-help-us-understand-rse-community. 
+
+**You can complete the 2021 RSE survey in
+[English](https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fsoftwaresaved.limequery.com%2F386272%3Flang%3Den&data=04%7C01%7Csjh%40ecs.soton.ac.uk%7C21dac81c36e14a07289308d9a81484de%7C4a5378f929f44d3ebe89669d03ada9d8%7C0%7C0%7C637725630602050621%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000&sdata=c50UYjuK%2FN0%2BYpxuKhOXWBJTyxpAkDhq5XTTamG%2FW%2Fc%3D&reserved=0),
```

Comment:
Is this wrapping of the original links in Outlook safelinks stuff intentional?

Reply:
I doubt it - good catch

Reply:
That's the way Simon had set it up. I'm going to ask him if it's intentional (he may know something I don't).

Reply:
I think it's just a cut and paste from an email - I can't imagine adding extra complexity to a known URL was intentional.

Reply:
Fixed it. Thanks for the catch @mtbc.